src/charm.py

```diff
@@ -91,14 +78,16 @@ def _validate_config(self):
 
     @property
     def monitoring_address(self):
-        binding = self.model.get_binding("monitoring")
+        binding = self.model.get_binding("metrics-endpoint")
         if not binding:
             return None
         return str(binding.network.ingress_address or binding.network.bind_address)
 
     @property
     def layer(self):
         """Pebble layer for workload."""
+        layer_args = list(self.config.keys())
```

Comment:
this excludes `scrape-interval` from showing up as a config to the metrics endpoint server